tensorflow/core/platform/default/logging.cc

```diff
@@ -109,14 +109,29 @@ int64 MinLogLevel() {
   }
 }
 
+int64 MinLogLevelFromEnv() {
+  const char* tf_env_var_val = getenv("TF_CPP_MIN_LOG_LEVEL");
+  return LogLevelStrToInt(tf_env_var_val);
+}
+
+int64 MinVLogLevelFromEnv() {
+  const char* tf_env_var_val = getenv("TF_CPP_MIN_VLOG_LEVEL");
```

Comment:
Oh, we need to add a TODO here that this currently doesn't work for values not in [1,2,3], so VLOG(4) statements will never print.

Ideally, we'd use safe_strto32 in core/lib/strings/numbers.h, but we don't want to add that dependency into the platform logging library since it would bring in StringPiece and a bunch of other code, so we'd need some simple, sanitizing version of string to integer.  I don't know, maybe std::stoi could be used here safely.